typescript/web/src/hooks/test-utils.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * This constant is not exported directly from
+ * `typescript/web/src/hooks/use-image-navigation.fixtures.ts`
+ * otherwise, the `mockNextRouter` function does not work, making all
+ * tests in `typescript/web/src/hooks/__tests__/use-images-navigation.tsx`
+ * fail
```

Comment:
It happens in many tests, we have to mock the router before some imports. In my case, the router has to be mocked before the mocked query in the `.fixtures.ts` file.
And because I wanted to use an imported variable in my mocked router, I had to extract it from my `.fixtures.ts` file